### ISSUE TITLE
accessibility support

### ISSUE THREAD

**Issue by nitedog** [w3c/strategy#56]
Collecting information about the support for [WCAG Techniques](https://www.w3.org/TR/WCAG20-TECHS/) in:
- web technologies
- browsers and media players
- assistive technology

Similar to platforms like [Can I use](http://caniuse.com/) but on entire Techniques rather than on browser implementation of specific technology features. For example, [Technique ARIA 9](https://www.w3.org/TR/WCAG20-TECHS/complete.html#ARIA9) does not only require the browser to implement "aria-labelledby" but also that a particular use of this attribute is supported by assistive technologies.

A [prototype "Accessibility Support Database"](https://www.w3.org/WAI/accessibility-support/) was previously developed through EU project funds. Concentrated effort would be needed to get crowd sourcing by individuals and organizations in the community.

**Comment by svgeesus** [comment]
Since caniuse is maintained on GitHub: https://github.com/Fyrd/caniuse
one option is to add the wcag techniques data to that. This would provide broader adoption, perhaps.

**Comment by nitedog** [comment]
@svgeesus agreed, and I believe we have the necessary APIs in our tool. The question is how can we get the snowball rolling on crowd-sourcing.

**Comment by dontcallmedom** [comment]
AFAICT there is active work in this space in the [ARIA and Assistive Technologies Community Group ](https://www.w3.org/community/aria-at/) - should this be moved to "Incubation"?
